packages/openchs-android/src/views/common/Observations.js

```diff
@@ -71,14 +72,34 @@ class Observations extends AbstractComponent {
             this.props.form.orderObservations(this.props.observations);
     }
 
-    renderValue(obs, isAbnormal, renderType) {
+    makeCall(number) {
+        RNImmediatePhoneCall.immediatePhoneCall(number);
+    }
+
+    renderValue(obs, isAbnormal, renderType, concept) {
         if ([Concept.dataType.Image, Concept.dataType.Video].includes(renderType)) {
             return (
                 <View style={this.styles.observationColumn}>
                     <ExpandableMedia source={obs} type={renderType}/>
                 </View>
             );
         }
+
+
+        const keyValue = concept.recordValueByKey('primary_contact') || concept.recordValueByKey('contact_number');
```

Comment:
Here also you can use the same model method as mentioned in comment https://github.com/avniproject/avni-client/pull/388/files#r499375067